docs/user_workloads/liveness_and_readiness_probes.md

```diff
@@ -313,71 +302,68 @@ cloud-init assigns the proper SELinux context, i.e. virt_qemu_ga_exec_t, to the
 
 ```yaml
 apiVersion: kubevirt.io/v1
-kind: VirtualMachineInstance
+kind: VirtualMachine
 metadata:
-  labels:
-    kubevirt.io/vm: vmi-guest-probe-vmi
-  name: vmi-fedora
+  name: example-agent-readiness-probe
 spec:
+  runStrategy: Always
   template:
-    metadata:
-      labels:
-        kubevirt.io/domain: vmi-guest-probe
-        kubevirt.io/vm: vmi-guest-probe
-  domain:
-    devices:
-      disks:
-      - disk:
-          bus: virtio
-        name: containerdisk
-      - disk:
-          bus: virtio
-        name: cloudinitdisk
-      rng: {}
-    resources:
-      requests:
-        memory: 1024M
-  readinessProbe:
-    exec:
-      command: ["cat", "/tmp/healthy.txt"]
-    failureThreshold: 10
-    initialDelaySeconds: 20
-    periodSeconds: 10
-    timeoutSeconds: 5
-  terminationGracePeriodSeconds: 180
-  volumes:
-  - name: containerdisk
-    containerDisk:
-      image: quay.io/containerdisks/fedora
-  - cloudInitNoCloud:
-      userData: |-
-        #cloud-config
-        password: fedora
-        user: fedora
-        chpasswd: { expire: False }
-        packages:
-          qemu-guest-agent
-        runcmd:
-          - ["touch", "/tmp/healthy.txt"]
-          - ["sudo", "chcon", "-t", "virt_qemu_ga_exec_t", "/tmp/healthy.txt"]
-          - ["sudo", "systemctl", "enable", "--now", "qemu-guest-agent"]
-    name: cloudinitdisk
+    spec:
+      domain:
+        devices:
+          disks:
+            - name: containerdisk
+              disk:
+                bus: virtio
+            - name: cloudinitdisk
+              disk:
+                bus: virtio
+          rng: {}
+        resources:
+          requests:
+            memory: 1Gi
+      readinessProbe:
+        exec:
+          command: ["cat", "/tmp/healthy.txt"]
+        failureThreshold: 10
+        initialDelaySeconds: 120
```

Comment:
From 20 to 120 seconds. Can I ask you the reason?

Reply:
There was a reason.. can't fully remember, but IIRC it was something along the lines of when I was running through the examples testing them myself (as an end user would) a longer period felt a lot better and more apparent to easily observe of what was happening.

Reply:
Thank you!